tools/generator/autorest_ext/metadata.go

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package autorest_ext
+
+import (
+	"github.com/Azure/azure-sdk-for-go/tools/generator/autorest"
+	"github.com/Azure/azure-sdk-for-go/tools/generator/autorest/model"
+	"github.com/Azure/azure-sdk-for-go/tools/generator/autorest_ext/model_ext"
+	"strings"
+)
+
+func GetAdditionalOptions(metadata autorest.GenerationMetadata) model.Options {
```

Comment:
We should have the same function in `tools/generator/autorest` too, could we use the functions there?
If there is not one in `tools/generator/autorest`, please feel free to move this function there

Reply:
fixed